index.html

```diff
@@ -0,0 +1,290 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Personal Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,400&family=Poppins:wght@300;400;700&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+        integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <script src="https://kit.fontawesome.com/38b6d4ab77.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <!-- navbar section -->
+        <a href="#" class="logo-class">My Logo</a>
+        <nav>
+            <ul class="navbar-item">
+                <li class="navbar-links-part"><a href="#portfolio-link">Portfolio</a></li>
+                <li class="navbar-links-part"><a href="#about-link">About</a></li>
+                <li class="navbar-links-part"><a href="#contact-link">Contact</a></li>
+            </ul>
+        </nav>
+        
+            <ul id="navbar-item-mobile">
+                <li><a href="#" class="navbar-links-partt nav-item" id="close-menu">X</a></li>
+                <li class="navbar-links-part nav-mobile"><a href="#portfolio-link" class="nav-a">Portfolio</a></li>
+                <li class="navbar-links-part nav-mobile"><a href="#about-link" class="nav-a">About</a></li>
+                <li class="navbar-links-part nav-mobile"><a href="#contact-link" class="nav-a">Contact</a></li>
+            </ul>
```

Comment:
While you made a great effort to make the font-sizes of the input items intuitively match the one that is provided by Figma design, you still have a slight change to make\

- [ ] Kindly reduce the margin/ padding top between each menu item. Currently, the spacing between each menu item is large and does not intuitively match the design provided. Having an almost identical design will ensure that your design looks professional improving the user experience of your portfolio. Remember this is the same portfolio you will be sharing to recruiters when looking for a job and making it look stellar should be a priority to you.

**This is how your design looks like:**

![image](https://github.com/sadaf-Daneshgar/First-Project-Microverse/assets/86318284/859d24d1-21ad-4ac0-99d0-d703781ad1a2)

**This is the desired design:**

![image](https://github.com/sadaf-Daneshgar/First-Project-Microverse/assets/86318284/03f9ca38-ef30-450e-a85b-b7d65d55bd7e)

**Note the differences in the vertical spaces between each menu item?**